internal/internal_workflow_testsuite.go

```diff
@@ -363,6 +363,7 @@ func (env *testWorkflowEnvironmentImpl) newTestWorkflowEnvironmentForChild(param
 	childEnv.testWorkflowEnvironmentShared = env.testWorkflowEnvironmentShared
 	childEnv.workerOptions = env.workerOptions
 	childEnv.workerOptions.DataConverter = params.dataConverter
+	childEnv.workflowInterceptors = env.workflowInterceptors
```

Comment:
This seems to be an issue for tests only. Please make it clear in PR title

Reply:
sure. For some reason my title got cut in PR and now I've fixed it.